src/api/rosetta-constants.ts

```diff
@@ -11,6 +12,15 @@ export const RosettaConstants = {
   rosettaVersion: '1.4.6',
   symbol: 'STX',
   decimals: 6,
+  lockedBalance: 'LockedBalance',
+  spendableBalance: 'SpendableBalance',
+};
+
+export const ReferenceNodes: { [key: string]: any } = {
+  mainnet: {
```

Comment:
Do we have testnet info for this too ?

Reply:
I've asked that [here](https://github.com/Zondax/stacks-blockchain-api/issues/25#issuecomment-791599981)
Still waiting for that info